apps/protocol-frontend/src/components/ReportForm.tsx

```diff
@@ -306,16 +307,18 @@ const ReportForm = ({ onFinish }: { onFinish: () => void }) => {
             label="DAO"
             tip={
               <>
-                Please select a DAO to associate this Contribution with. This is
-                optional. Dont see your DAO? Request to add it{' '}
-                <a
-                  href="https://airtable.com/shrOedOjQpH9xlg7l"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  style={{ textDecoration: 'underline' }}
-                >
-                  here
-                </a>
+                Please select a DAO to associate this Contribution with. 
+                <Box fontWeight={700}>
```

Comment:
We may want to either add a tiny bit of padding or increase the line-height. We can play with this to see what we think

Reply:
Going to push up a quick change to use Chakra's link and also some prettier fixes